escrow/tests/Spec/Basic.hs

```diff
@@ -0,0 +1,37 @@
+module Spec.Basic (tests) where
+
+import Test.Tasty
+import Test.Tasty.QuickCheck as QC
+import Test.Tasty.HUnit
+import Plutus.Contract.Test
+
+import Data.List
+import Data.Ord
+
+testWallets :: [Wallet]
+testWallets = [w1, w2, w3, w4, w5]
```

Comment:
https://github.com/aleeusgr/minimal-ptt-examples/issues/18

https://input-output-hk.github.io/plutus-apps/main/plutus-contract/html/Wallet-Emulator-Wallet.html#t:Wallet

https://wiki.haskell.org/Constructor

Reply:
https://input-output-hk.github.io/plutus-apps/main/plutus-contract/html/Wallet-Emulator-Wallet.html

Reply:
:t w1

Reply:
```haskell
Prelude Plutus.Contract.Test Spec.Basic> :doc w1
ghc: Can't find any documentation for Plutus.Contract.Test.
This is probably because the module was compiled without '-haddock',
but it's also possible that the module contains no documentation.

```